src/molecules/Modal/Modal.tsx

```diff
@@ -0,0 +1,44 @@
+// Generated with util/create-component.js
+import React from "react";
+import ReactModal from "react-modal";
+import { useTheme } from "styled-components";
+
+import { ModalProps } from "./Modal.types";
+
+const Modal: React.FC<ModalProps> = ({
+    open,
+    color = "primary",
+    onOverlayClick,
```

Comment:
thoughts on making this align with the props being passed into ReactModal and call it `onClose` or `onRequestClose`

Reply:
Changed to `onRequestClose`